changelogs/unreleased/5466-ensure-that-API-endpoints-sort-their-results-in-a-consistent-way.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: "The following API endpoints now return their results in a meaningful order: methods.list_settings, methods_v2.environment_settings_list, methods.list_params, methods_v2.get_facts, methods.list_projects, methods_v2.project_list."
+change-type: patch
+destination-branches: [master, iso6, iso5]
```

Comment:
I pondered adding one for a while but decided against it because I felt like this was not that important to the end user. I don't mind adding one.
This would be a `minor-improvement` ? And is the current description ok to go in as-is, or is it too specific ?

Reply:
I think this should go into the bugfix section actually. The current description looks fine to me.